### PR TITLE
Big bug fix

### LINKED ISSUE #602: Use cleanContent instead of content for commands like ASCIIfy and cowsay
IF you mention someone/a channel in `s!asciify`, the command gets the internal representation of the mention instead of its text, resulting in some borks

### LINKED ISSUE #603: s!time has a missing string
image

@SwitchbladeBot/translation-managers

### LINKED ISSUE #615: You can use 0, negative and decimal numbers to remove a song from the queue
**Bug description**
You can use 0, negative and decimal numbers to remove a song from the queue.

**How to reproduce**
1. Run c!queue remove 0/1.1/-3

**Expected behavior**
The bot should give and error.

**Screenshots**
screenshot_20190127195002
screenshot_20190127195013
screenshot_20190127195020

### LINKED ISSUE #616: You can go to a negative part of a song using c!seek
**Bug description**
You can go to a negative part of a song using c!seek.

**How to reproduce**
1. Run c!seek -30 when the song just started.

**Expected behavior**
The bot should give an error.

**Screenshots**
screenshot_20190127195133

### LINKED ISSUE #617: c!seek can't always go to end of song
**Bug description**
c!seek can't always go to end of song.

**How to reproduce**
1. Run c!seek to go to the end of a song
2. Run on another song.

**Expected behavior**
The playback should go to the end of the song.

**Screenshots**
screenshot_20190127200125

### LINKED ISSUE #618: Bug in DBL command
**Bug description**
When executing the command * * DBL * * It automatically gives 1k200 of coins even not having voted...

**How to reproduce**
1. Run c!dbl

**Expected behavior**
When the person has voted he gives you a certain reward

**Screenshots**
http://prntscr.com/mcx3rk

### LINKED ISSUE #619: Bug in c!daily
**Bug description**
I had not taken my daily and when I went to run the command he said I had already caught...

**How to reproduce**
1. Run c!daily

**Expected behavior**
When you run the command it gives you a reward and only after 24hrs it gives you again

**Screenshots**
http://prntscr.com/mcx6rs

### LINKED ISSUE #620: Bug on TTT
**Bug description**
You can play with yourself

**How to reproduce**
1. Run c!ttt @mention

**Expected behavior**
Play with other people and not with yourself

**Screenshots**
http://prntscr.com/mcxgjj

### LINKED ISSUE #621: Bug on birb command
**Bug description**
The bird command is no longer working

**How to reproduce**
1. Run s!bird

**Expected behavior**
This command to work

### LINKED ISSUE #622: You can use 0, negative and decimal numbers to jump to a song in the queue
**Bug description**
You can use 0, negative and decimal numbers to jump to a song in the queue.

**How to reproduce**
1. Run c!queue jump 0/1.1/-3

**Expected behavior**
The bot should give and error.

### LINKED ISSUE #623: Jumping to a song while playback is looped removes the songs before the song oneself jumped to
**Bug description**
Jumping to a song while playback is looped removes the songs before the song oneself jumped to.

**How to reproduce**
1. Add songs to the queue.
2. Start looping playback with c!loop.
3. Jump to a song using c!queue jump <number>.

**Expected behavior**
The songs should stay.

**Screenshots**
screenshot_20190127224017

### LINKED ISSUE #624: c!queue subcommand strings are displaying as undefined
**Bug description**
c!queue subcommand strings are displaying as undefined.

**How to reproduce**
Steps to reproduce the unusual behavior:
1. Run c!queue remove or c!queue jump

**Expected behavior**
The strings (if there are any) should display correctly.

**Screenshots**
screenshot_20190127224222

### PR DESCRIPTION
**Pull request by dpaiv0**
+ Fixes #602 
+ Fixes #603 
+ Fixes #615
+ Fixes #616
+ Fixes #617 
+ Fixes #618
+ Fixes #619
+ Fixes #620
+ Fixes #621
+ Fixes #622 
+ Fixes #623
+ Fixes #624

### CODE REVIEW

File: src/commands/music/seek.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ module.exports = class Seek extends Command {
 
     const position = guildPlayer.state.position + target
     if (position > playingSong.length) throw new CommandError(t('commands:seek.lengthExceeded'))
+    if (position < 0) throw new CommandError(t('commands:seek.noNegativeValues'))
```
Comment (davipatury) [inline src/commands/music/seek.js]:
```suggestion
    if (position <= 0) throw new CommandError(t('commands:seek.noNegativeValues'))
```

File: src/structures/command/parameters/types/NumberParameter.js [hunk 2]
```diff
@@ -19,8 +19,9 @@ module.exports = class NumberParameter extends Parameter {
       throw new CommandError(t('errors:invalidNumber'))
     }
 
-    if (!isNull(this.min)) nmb = Math.max(nmb, this.min)
-    if (!isNull(this.max)) nmb = Math.min(nmb, this.max)
+    if (!!this.min && nmb < this.min) throw new CommandError(t('errors:needBiggerNumber', { number: this.min }))
+    if (!!this.max && nmb > this.max) throw new CommandError(t('errors:needSmallerNumber', { number: this.max }))
```
Comment (davipatury) [inline src/structures/command/parameters/types/NumberParameter.js]: `!!0 === false`
If the minimum or maximum value is set to 0 it won't work.

**Comment by pedrofracassi** [comment]
This is beautiful

File: src/commands/misc/asciify.js [hunk 3]
```diff
@@ -13,8 +13,8 @@ module.exports = class Asciify extends Command {
     )
   }
 
-  run ({ channel }, text) {
-    const bigtext = figlet.textSync(text, {
+  run ({ channel, message }) {
+    const bigtext = figlet.textSync(message.cleanContent.split(' ').slice(1).join(' '), {
```
Comment (pedrofracassi) [inline src/commands/misc/asciify.js]: Splitting by 1 can still cause problems on servers that use custom prefixes that contain spaces. I suggest making a "CleanStringParameter"

File: src/commands/misc/cowsay.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ module.exports = class Cowsay extends Command {
     )
   }
 
-  run ({ channel }, text) {
-    channel.send(`\`\`\`${cowsay.say({ text })}\`\`\``)
+  run ({ channel, message }) {
+    channel.send(`\`\`\`${cowsay.say({ text: message.cleanContent.split(' ').slice(1).join(' ') })}\`\`\``)
```
Comment (pedrofracassi) [inline src/commands/misc/cowsay.js]: Same as above

File: src/commands/misc/asciify.js (hunk 3 above)
Comment (pedrofracassi) [reply src/commands/misc/asciify.js]: Sounds good

**Comment by metehus** [comment]
wow

File: src/commands/misc/asciify.js (hunk 3 above)
Comment (davipatury) [reply src/commands/misc/asciify.js]: Fixed by #629

**Pull request merged**